### PR TITLE
Wrangelkiez story new

### PR DESCRIPTION
**Pull request by Lisa-Stubert**

### CODE REVIEW

**Review by ff6347 — APPROVED** [review]
LGTM

**Review by Esshahn — APPROVED** [review]
LGTM with G being GREAT

File: src/wrangelkiez/index.html [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+                  return data
+
+                sensor_id = '22' # define sensor id here, this is Albert
+                time_begin = '2021-11-18' # define the timerange
+                time_end = '2021-11-24'
+                data = fetch("https://api.stadtpuls.com/api/v3/sensors/"+sensor_id+"/records")
+                data = pd.DataFrame.from_dict(data) 
+                data = data[time_begin:time_end] # extract only measurements in the timerange
+              </code>
+              Den kompletten Python-Code zur Verarbeitung der Daten findest du in <a style="color:#8330ff;" href="https://deepnote.com/project/Stadtpuls-Story-Larm-svXmxLJuSAGiqoQitfBt3w/%2Fnotebook.ipynb/#00008-adf6dbfa-d0dd-4fc2-8a85-eea3b050dd7e">Deepnote</a>.
```
Comment (ff6347) [inline src/wrangelkiez/index.html:305]: Hm. Should this be within the <pre> tag? Or better in a <p> below?

**Review by ff6347 — CHANGES_REQUESTED** [review]

**Review by ff6347 — APPROVED** [review]
AWESOMEEEEEEEE! 🚀

**Comment by Esshahn** [comment]
whoohoooo!

**Pull request merged**